src/containerapp/azext_containerapp/_params.py

```diff
@@ -209,6 +211,10 @@ def load_arguments(self, _):
         c.argument('name', id_part=None)
         c.argument('revision', help='Name of the revision.')
         c.argument('label', help='Name of the label.')
+        c.argument('yes', options_list=['--yes', '-y'], help='Do not prompt for confirmation.')
+
+    with self.argument_context('containerapp revision label') as c:
+        c.argument('labels', nargs='*', help='Labels to be swapped.')
```

Comment:
--labels can be confusing - & might need exta error handling. Please update this to -source-label -target-label.